apps/o2-blocks/src/github-issue-template/editor.js

```diff
@@ -31,14 +30,10 @@ const githubIcon = (
 const Shell = ( { as: El = 'div', className, title, subTitle, icon, body, ...elementProps } ) => {
```

Comment:
Another nit-pick from me 🙈: this can be `subtitle` as it's one word.

Reply:
👍🏻